components/combobox/combobox.jsx

```diff
@@ -615,6 +623,16 @@ class Combobox extends React.Component {
 			options.push(...localProps.optionsSearchEntity);
 		}
 
+		if (props.hasDeselect) {
+			// eslint-disable-next-line fp/no-mutating-methods
```

Comment:
instead of continuing the anti pattern that requires adding the eslint exception why not create methods that create the options object with the additional option. that way the individual methods can be united tested.